.golangci.yml

```
@@ -22,9 +22,11 @@ linters:
       branch-max-lines: 2
 formatters:
   enable:
+    - gci
     - gofmt
-    - goimports
   settings:
-    goimports:
-      local-prefixes:
-        - github.com/statnett/image-scanner-operator
+    gci:
+      sections:
+        - standard # Standard section: captures all standard packages.
+        - default # Default section: contains all imports that could not be matched to another section type.
+        - prefix(github.com/statnett/image-scanner-operator) # Custom section: groups all imports with the specified Prefix.
```

Or is it a reason for upper-case?
```suggestion
        - prefix(github.com/statnett/image-scanner-operator) # Custom section: groups all imports with the specified prefix.
```